docs/actions/bundle_install.md

```diff
@@ -25,28 +25,29 @@ Author | @birmacher, @koglinjg
-  `full_index` | Use the rubygems modern index instead of the API endpoint
-  `gemfile` | Use the specified gemfile instead of Gemfile
-  `jobs` | Install gems using parallel workers
-  `local` | Do not attempt to fetch gems remotely and use the gem cache instead
-  `deployment` | Install using defaults tuned for deployment and CI environments
-  `no_cache` | Don't update the existing gem cache
-  `no_prune` | Don't remove stale gems from the cache
-  `path` | Specify a different path than the system default ($BUNDLE_PATH or $GEM_HOME). Bundler will remember this value for future installs on this machine
-  `system` | Install to the system location ($BUNDLE_PATH or $GEM_HOME) even if the bundle was previously installed somewhere else for this application
-  `quiet` | Only output warnings and errors
-  `retry` | Retry network and git requests that have failed
-  `shebang` | Specify a different shebang executable name than the default (usually 'ruby')
-  `standalone` | Make a bundle that can work without the Bundler runtime
-  `trust_policy` | Sets level of security when dealing with signed gems. Accepts `LowSecurity`, `MediumSecurity` and `HighSecurity` as values
-  `without` | Exclude gems that are part of the specified named group
-  `with` | Include gems that are part of the specified named group
-
-
+Key | Description | Default
+----|-------------|--------
+  `binstubs` | Generate bin stubs for bundled gems to ./bin | -
+  `clean` | Run bundle clean automatically after install | false
+  `full_index` | Use the rubygems modern index instead of the API endpoint | false
+  `gemfile` | Use the specified gemfile instead of Gemfile | -
+  `jobs` | Install gems using parallel workers | -
+  `local` | Do not attempt to fetch gems remotely and use the gem cache instead | false
+  `deployment` | Install using defaults tuned for deployment and CI environments | false
+  `no_cache` | Don't update the existing gem cache | false
+  `no_prune` | Don't remove stale gems from the cache | false
+  `path` | Specify a different path than the system default ($BUNDLE_PATH or $GEM_HOME). Bundler will remember this value for future installs on this machine | -
+  `system` | Install to the system location ($BUNDLE_PATH or $GEM_HOME) even if the bundle was previously installed somewhere else for this application | false
+  `quiet` | Only output warnings and errors | false
+  `retry` | Retry network and git requests that have failed | -
+  `shebang` | Specify a different shebang executable name than the default (usually 'ruby') | -
+  `standalone` | Make a bundle that can work without the Bundler runtime | -
+  `trust_policy` | Sets level of security when dealing with signed gems. Accepts `LowSecurity`, `MediumSecurity` and `HighSecurity` as values | -
+  `without` | Exclude gems that are part of the specified named group | -
+  `with` | Include gems that are part of the specified named group | -
+
+_- = this parameter doesn't have a default value_<br/>
```

Comment:
to get rid of the "this":

Reply:
- = no default value for this parameter
* = default value is dependent on the user's system